TDDStrictequal.js

```diff
@@ -1,5 +1,5 @@
 const strictEquals = (valNumOne, valNumTwo) => {
-
+  return typeof valNumOne == typeof valNumTwo && valNumOne == valNumTwo
 }
```

Comment:
Very clean implementation. Of course since this is a one line function with only a return we can use fat arrow's implicit return

```JavaScript
const strictEquals = (valNumOne, valNumTwo) => typeof valNumOne == typeof valNumTwo && valNumOne == valNumTwo
```